terraform/testcases/xrayreceiver/parameters.tfvars

```diff
@@ -3,7 +3,7 @@ validation_config = "default-xray-trace-validation.yml"
 
 # sample application image to emit the trace data
 # src: https://github.com/aws/aws-xray-sdk-go/blob/master/sample-apps/http-server/application.go
-sample_app_image = "611364707713.dkr.ecr.us-west-2.amazonaws.com/amazon/aws-otel-goxray-sample-app:v1.2.0"
+sample_app_image = "public.ecr.aws/aws-otel-test/aws-otel-goxray-sample-app:v1.2.0"
```

Comment:
This has arm support?

Reply:
Currently not. But we can build it from the source code with same tags though.